.github/workflows/server_compatibility.yaml

```
@@ -96,11 +99,20 @@ jobs:
           name: hazelcast-enterprise
           path: hazelcast-enterprise/hazelcast-enterprise/target/hazelcast-enterprise-*[!s].jar
           retention-days: 1
-      - name: Upload Hazelcast Enterprise tests JAR
+      - name: Rename certs.jar as Hazelcast Enterprise tests JAR
+        # - Find a file named "hazelcast*-tests.jar" in the $GITHUB_WORKSPACE/hazelcast/hazelcast/target -> find command
+        # - Get rid of the file path, so that we only have the file name -> piped sed command
+        # - Replace the hazelcast with hazelcast-enterprise so that we can end up with the expected
+        #   enterprise test file name -> parameter expansion
+        # - Rename the certs.jar with that -> mv command
+        run: |
+          mv certs.jar "${$(find $GITHUB_WORKSPACE/hazelcast/hazelcast/target -maxdepth 1 -mindepth 1 -name "hazelcast*" -a -name "*tests.jar" | sed "s/.*\///")//hazelcast/hazelcast-enterprise}"
```

Did not get this command fully.

what about this? 

```suggestion
          mv certs.jar $(basename $(ls hazelcast/target/hazelcast-*-tests.jar))
```

basename exists by default in ubuntu.

---

Anyway, I am ok if it works. Can we try to run all backward compatilbity tests once? (server and all clients)

---

Let's try this as it is, and if it does not work we can try yours. The comment above should explain what each part does